src/main/java/net/milkbowl/vault/economy/Economy.java

```diff
@@ -89,12 +88,12 @@ public interface Economy {
      * Checks if this player has an account on the server yet
      * This will always return true if the player has joined the server at least once
      * as all major economy plugins auto-generate a player account when the player joins the server
-     * 
+     *
      * @param player to check
      * @return if the player has an account
      */
     public boolean hasAccount(OfflinePlayer player);
-    
+
```

Comment:
An example of a whitespace diff.

If this were to be merged, the git blame/history will be chopped up all over of the place. You can see there are many unneeded changes: https://github.com/MilkBowl/VaultAPI/pull/152/files